UIKitDZ/FirstTrackViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  FirstTrackViewController.swift
+//  UIKitDZ
+//
+//  Created by coder on 25.09.2022.
+//
+import AVFoundation
+import UIKit
+/// FirstTrackViewController - экран с первым треком
+class FirstTrackViewController: UIViewController {
+    
+    var player = AVAudioPlayer()
+    
+    @IBOutlet weak var imageFirstTrack: UIImageView!
+    @IBOutlet weak var sliderVolume: UISlider!
+    @IBAction func sliderVolumeAction(_ sender: UISlider) {
+        player.volume = sliderVolume.value
+    }
+    
+//    @IBAction func forwardAction(_ sender: UIButton) {
+//        guard let newVc = storyboard?.instantiateViewController(withIdentifier: "vc3")
+//                as? SecondTrackViewController else { return }
+//        self.present(newVc, animated: true)
+//    }
+    @IBOutlet weak var slider: UISlider!
+    @IBOutlet weak var labelTimeTrack: UILabel!
```

Comment:
```suggestion
    @IBOutlet weak var timeTrackLabel: UILabel!
```
и везде ниже по такому принципу переименовать